### PR TITLE
Monit configuration respects options_per_process

### PR DESCRIPTION
**Pull request by kazjote**

### CODE REVIEW

File: lib/capistrano/tasks/monit.rake [hunk 1]
```diff
@@ -119,6 +119,10 @@ namespace :sidekiq do
       end
     end
 
+    def sidekiq_options_per_process
+      fetch(:sidekiq_options_per_process) || []
```
Comment (seuros) [inline lib/capistrano/tasks/monit.rake]: fetch accepts a second argument that is returned if the key is nil
```
fetch(:sidekiq_options_per_process,[])
```

**Comment by seuros** [comment]
Can you update the readme ?

**Comment by christoph-buente** [comment]
It is just passing the same options_per_process to the monitor start stop commands that are passed to the initial start command. It is plainly missing in the monit template.

**Comment by seuros** [comment]
Thanks you :green_heart:

**Comment by kazjote** [comment]
Thanks for maintaining this great project! :+1:

**Pull request merged**